src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -148,10 +149,14 @@ export const CollapsibleNavTop = ({
           />
         </EuiFlexItem>
       </EuiFlexGroup>
-      {currentNavGroup?.title && (
+      {(currentNavGroup?.type || collapsibleNavHeaderRender) && (
```

Comment:
@SuZhou-Joe Is this type or title?

Reply:
Add some comment and optimize the code here to reduce confusion.